packages/snowpack/example/babel.config.json

```diff
@@ -1,7 +1,18 @@
 {
-  "presets": [["@babel/preset-react", {
-    "pragma": "h",
-    "pragmaFrag": "Fragment"
-  }], "@babel/preset-typescript"],
-  "plugins": ["@babel/plugin-syntax-import-meta", "react-refresh/babel"]
+  "presets": [
+    [
+      "@babel/preset-react",
+      {
+        "pragma": "h",
+        "pragmaFrag": "Fragment"
+      }
+    ],
+    "@babel/preset-typescript"
+  ],
+  "plugins": ["@babel/plugin-syntax-import-meta"],
+  "env": {
+    "development": {
+      "plugins": ["react-refresh/babel"]
+    }
```

Comment:
Minor change, but moved this `react-refresh/babel` into a development-only config because it throws an error on `snowpack build`